stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -781,13 +784,15 @@ pub struct SortitionDB {
 pub struct SortitionDBTxContext {
     pub first_block_height: u64,
     pub pox_constants: PoxConstants,
+    pub dryrun: bool,
```

Comment:
Can you add a rustdoc for the new `dryrun` fields explaining that they are only set to true in testing or by the analysis tools in `stacks-inspect` and what their impact is?

Reply:
Yup sure